src/Modules/ModulesGraded.jl

```diff
@@ -414,7 +414,7 @@ function set_grading!(M::FreeMod, W::Vector{<:IntegerUnion})
   M.d = [W[i] * A[1] for i in 1:length(W)]
 end
 
-function degrees(M::FreeMod)
+function degrees(M::FreeMod; check::Bool=true)
```

Comment:
What is this `check` argument for? I am asking because it is not used in the function body. Is it just for consistency with other module types?